presto-kudu/pom.xml

```diff
@@ -167,4 +167,56 @@
             </plugins>
         </pluginManagement>
     </build>
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <excludes>
+                                <exclude>**/TestKuduIntegrationDecimalColumns.java</exclude>
```

Comment:
How about
```java
<exclude>**/TestKuduIntegration*.java</exclude>

Reply:
Maybe we could have package-based exclusion?

Reply:
It looks like packages are possible: https://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html

Maybe they have a heuristic to know if it's a path vs package?

Another option is to use TestNG groups.